magento/pwa-studio
# [PWA-623]: Update Venia's inputs

## Description

Adjust input styles to match the new design

## Related Issue
https://jira.corp.magento.com/browse/PWA-623

## Acceptance 
* Verify inputs are updated throughout the app
* Ensure there is no broken functionality with the styling change

### Verification Stakeholders
@soumya-ashok 
@jimbo 
@dpatil-magento 

### Specification



### Verification Steps


1. Navigate around the app following all known user flows
2. Verify all inputs previously present are present and work
3. Verify all inputs are positioned and relevant labels are readable

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
